src/main/scala/com/microsoft/ml/spark/lightgbm/LightGBMBooster.scala

```diff
@@ -8,38 +8,45 @@ import com.microsoft.ml.spark.lightgbm.LightGBMUtils.getBoosterPtrFromModelStrin
 import org.apache.spark.ml.linalg.{DenseVector, SparseVector, Vector}
 import org.apache.spark.sql.{SaveMode, SparkSession}
 
+class BoosterHandler(model: String) extends Serializable {
+
+  LightGBMUtils.initializeNativeLibrary()
+
+  lazy val boosterPtr: SWIGTYPE_p_void = {
```

Comment:
I think you still want these fields to have `@transient` so they are excluded from spark task serialization

Reply:
Could be, but isn't it a little redundant if I declare the instance of this class to be transient?

Reply:
Initially I was confused because the class was still serializable, but I see you've fixed that now. 👍

Reply:
Yes, thanks, I realised it was dangerous, now if somebody wants to use it is forced to use transient, and that is why I thought it was a little redundant. I will remove now the inner transients.